HACKING.md

```diff
@@ -84,16 +85,16 @@ switch the Core Selector to “`release`” and click on “Query” to get to
 <http://localhost:8983/solr/#/release/query>.
 
 Then start filling the form as follows:
-* Set the first field “Request-Handler (qt)” (qt)
+* Set the first field `Request-Handler (qt)`
   to the value for `path` in the log record, with a leading slash:
   `/advanced`
-* Set the second field “q” (query string)
+* Set the second field `q` (query string)
```

Comment:
Because it’s a field name, I stuck the English double quotation marks as above.

I also thought that reserving code markup for values would make easier to spot and copy these.

It was not completely consistent though.

Reply:
Ok, we can also do that instead - I used backticks here because you used them for `wt` and whatnot elsewhere :) As long as it is consistent, it doesn't matter to me which one it is :)